README.md

```diff
@@ -40,126 +40,163 @@ worker:
+ `invenio.sentry.enabled`                            | Enable Sentry logging                                                                                             | `false`               
+ `invenio.sentry.existing_secret`                    | Whether to use an existing secret or create a new one                                                             | `false`               
+ `invenio.sentry.secret_name`                        | Name of the secret to use or create                                                                               | `sentry-secrets`      
+ `invenio.sentry.dsn`                                | DSN for sentry                                                                                                    | `""`                  
+ `invenio.datacite.enable`                           | Enable DataCite provider                                                                                          | `false`               
+ `invenio.datacite.existing_secret`                  | Whether to use an existing secret or create a new one                                                             | `false`               
+ `invenio.datacite.secret_name`                      | Name of the secret to use or create                                                                               | `datacite-secrets`    
+ `invenio.remote_apps.enabled`                       | Enable logging with remote applications                                                                           | `false`               
+ `invenio.remote_apps.existing_secret`               | Whether to use an existing secret or create a new one                                                             | `false`               
+ `invenio.remote_apps.secret_name`                   | Name of the secret to use or create                                                                               | `remote-apps-secrets` 
+ `invenio.remote_apps.credentials`                   | List of remote applications' credentials (name, consume_key, consumer_secret)                                     | `""`                  
+ `invenio.extra_config`                              | List of key:value configuration variables accepted by the python application                                      | `""`                  
+ `invenio.file_config_maps`                          | List of objects describing additional config maps read from file                                                  | `""`                  
+ `invenio.file_config_maps.<configMapName>.fileName` | Name of the file config map                                                                                       | `""`                  
+ `invenio.file_config_maps.<configMapName>.file`     | File content (should be loaded by --set-file invenio.file_configs.<configMapName>.file=<path-to-my-file>  option) | `""`                  
```

Comment:
```suggestion
 `invenio.file_config_maps.<configMapName>.file`     | File content (should be loaded by --set-file invenio.file_config_maps.<configMapName>.file=<path-to-my-file>  option) | `""`                  
```

comment: shouldn't these match? I see in `values.yaml` the name is used, but not sure if it's in a different context.